src/Features/Encoding/Braille/BrailleReference.test.tsx

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import SemaphoreTable from './SemaphoreTable';
+import BrailleReference from './BrailleReference';
```

Comment:
I'm guessing something wonky here with the diff-er.

Reply:
Yeah, seems like the renames confused it.